portfolio/src/main/webapp/index.html

```diff
@@ -23,12 +25,24 @@ <h1>Tal's Portfolio</h1>
             <li> I love traveling in Israel and around the world</li>
         </ul> 
 
-            Click here to get a random picture of a place in Israel I recently visited:
-            <button onclick="addRandomPic()"><b>GET A PIC</b></button>
-            <div id="image-container" > </div>
-            <div id= "location-title">
-            <p id="location-name"></p> </div>
-        
+        Click here to get a random picture of a place in Israel I recently visited:
```

Comment:
I think this block belongs to the Maps PR, right?

Reply:
Yes, I accidently started the charts branch from the maps branch and not from master as we discussed, and only after I finished I realized this. So I did the reset and stash trick to move the charts commits to be from master, but it got a little messy and somehow this remained

Reply:
Always `git log` :)